misc/crawl_db.js

```diff
@@ -0,0 +1,72 @@
+var Crawler = require('../src/crawler.js').Crawler
+var _ = require('lodash');
+var fs = require('fs');
+var Sequelize = require('sequelize');
+var Promise = Sequelize.Promise;
+var modelsFactory = require('../src/models.js');
+
+/* --------------------------------- CONSTS --------------------------------- */
+
+var config = {};
+
+['DROP_TABLES', 'LOG_SQL', 'LOG_CRAWL'].forEach(function(k) {
+  config[k] = process.env[k] !== undefined;
+});
```

Comment:
Here's a really useful module that can extract configuration stuff like this from environment variables automagically:

https://github.com/indexzero/nconf